models/agent_model.go

```diff
@@ -631,6 +647,32 @@ func (s Agent) TemplateDelimiters(svc *Service) *DelimiterPair {
 	return &tdp
 }
 
+// HashPassword func to calculate password hash
+var HashPassword = func(password, salt string) string {
```

Comment:
Why do we want to make HashPassword overridable and public?
As far as I remember we use static agent IDs in all tests.

Reply:
Because bcrypt hash function has quite high cyclomatic complexity which makes race tests run very long
I'm overriding that in tests to return just a plain password

Reply:
got it, thanks